src/PrestaShopBundle/Utils/ZipManager.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace PrestaShopBundle\Utils;
+
+use RecursiveIteratorIterator;
+use RecursiveDirectoryIterator;
+
+class ZipManager
+{
+    public function createArchive($filename, $folder)
```

Comment:
It would be great to make this function return something, or handle the exceptions by the caller.
If we do not do that and an error occurs, I wonder if the merchant will get a blank page.